src/style.css

```diff
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf),
+    url(fonts/Avenir-Heavy.ttf);
+}
+
+:root {
+  --color-defult: #3d4e61;
+  --color-box__shadow: #5c5c5c5b;
+}
+
+body {
+  font-family: "Avenir", serif;
+  font-weight: 300;
+}
+
+.search {
+  border: none;
+  background: none;
+  outline: none;
+  padding: 0;
+  width: 100%;
+  font-size: 16px;
+}
+
+.search::placeholder {
+  color: #000;
+  font-family: "Avenir", serif;
+}
+
+.search--form {
+  display: flex;
+  justify-content: center;
+  border: 1px solid #e1e7ed;
+  border-radius: 5px;
+  margin-top: 20px;
+}
+
+.search--form:focus-within {
+  box-shadow: 0 5px 10px var(--color-box__shadow);
+  color: #3d4e61;
+  font-weight: 700;
+}
+
+.search--form__small {
+  height: 42px;
+  padding-left: 33px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 13px 15px;
+  background-size: 11px;
+}
+
+.search--form__big {
+  height: 70px;
+  padding-left: 62px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 25px 26px;
```

Comment:
Do not hard code height - use paading instead
Do not repeat your self - move repeatable properties in  `.search` selector